docs/website/src/site/index.md

```diff
@@ -51,4 +51,120 @@ eleventyNavigation:
+<path class="ds-explore__components-part" d="M118.209 105.558L121.568 111.638" stroke="#5985FF" stroke-width="5.25962" stroke-miterlimit="10" stroke-linecap="round"/>
+<path class="ds-explore__components-part" d="M125.346 84.7039L131.64 81.7661" stroke="#5985FF" stroke-width="5.25962" stroke-miterlimit="10" stroke-linecap="round"/>
+<path class="ds-explore__components-part" d="M124.954 95.9535L133.071 98.6218" stroke="#5985FF" stroke-width="5.25962" stroke-miterlimit="10" stroke-linecap="round"/>
+</svg>
+
+
+</div>
+
+<div class="ds-heading-2">
+<a href="/categories/components/overview/" aria-label="Components. Sanoma Learning’s UI components are built to best practices to ensure usability, performance and accessibility.">
+Components
+</a>
+</div>
+
+Sanoma Learning’s UI components are built to best practices to ensure usability, performance and accessibility.
+
+</div>
+
+<div class="ds-explore__design-tokens ds-explore__card">
+
+<div class="ds-explore__card-image">
+
+<svg width="100" height="100" viewBox="0 0 100 100" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path class="ds-explore__design-tokens-dot" d="M48.718 92.3077C48.718 96.5559 45.2739 100 41.0257 100C36.7773 100 33.3334 96.5559 33.3334 92.3077C33.3334 88.0595 36.7773 84.6154 41.0257 84.6154C45.2739 84.6154 48.718 88.0595 48.718 92.3077Z" fill="#3366FF"/>
+<path class="ds-explore__design-tokens-dot" d="M74.359 92.3077C74.359 96.5559 70.9149 100 66.6667 100C62.4185 100 58.9744 96.5559 58.9744 92.3077C58.9744 88.0595 62.4185 84.6154 66.6667 84.6154C70.9149 84.6154 74.359 88.0595 74.359 92.3077Z" fill="#45CC80"/>
+<path class="ds-explore__design-tokens-dot" d="M100 92.3077C100 96.5559 96.5559 100 92.3077 100C88.0595 100 84.6154 96.5559 84.6154 92.3077C84.6154 88.0595 88.0595 84.6154 92.3077 84.6154C96.5559 84.6154 100 88.0595 100 92.3077Z" fill="#FF575C"/>
+<path d="M64.6508 79.6446L58.3333 64.2308H16.6667L7.17949 87.6923C6.49574 89.4015 5.3419 90.2564 3.71795 90.2564C2.52138 90.2564 1.62395 89.9574 1.02564 89.359C0.427334 88.7605 0.0854872 87.9487 0 86.9231V86.5385C0 86.0256 0.0427184 85.641 0.128206 85.3846L33.9744 2.5641C34.6581 0.854718 35.9402 0 37.8205 0C38.6754 0 39.4015 0.256411 40 0.769231C40.6836 1.19656 41.1964 1.79487 41.5385 2.5641L73.921 81.7359C71.8585 80.3174 69.3595 79.4872 66.6667 79.4872C65.9805 79.4872 65.3072 79.541 64.6508 79.6446ZM37.3077 12.8205L19.6154 56.6667H55.2564L37.3077 12.8205Z" fill="#3366FF"/>
+</svg>
+
+</div>
+
+<div class="ds-heading-2">
+<a href="/categories/design-tokens/" aria-label="Design tokens. Tokens are platform-agnostic variables that represent the look and feel of our brands.">
+Design tokens
+</a>
+</div>
+
+Tokens are platform-agnostic variables that represent the look and feel of our brands.
```

Comment:
maybe "define" or "make up" instead of represent?
That way it's clearer that the tokens are the input for the DS, so it looks the way you want.

Also maybe, use "your brand" instead of "our brands" that way you get the reader more involved

Reply:
@Diaan thanks for your input
@anna-lach can we change it to: 'Tokens are platform-agnostic variables that define the look and feel of your brand.